assets/js/components/icons/woo.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { Icon } from '@wordpress/components';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+const IconWoo = ( { size = 20, className } ) => (
+	<Icon
+		className={ classNames( 'woo-icon', className ) }
+		icon={
+			<svg
+				xmlns="http://www.w3.org/2000/svg"
+				height={ size }
```

Comment:
You should probably also set `width` here as well, or pass `size` to `Icon`. `Icon` defaults a size of 24, and sets both width and height on the SVG. So right now, if you do `<IconWoo  size={ 50 } />`, you'll get `height="50" width="24"` on the svg.

Reply:
Is there a way to avoid that? The icon is not square. When both height and width are set, the icon uses the smaller dimension and looks bad in the UI. The components seem to pass a size even if you try to override it when using this component.

Reply:
Yeah, I was thinking you could use the icon's ratio to calc the correct width – like, if it should be 20x24, do `height={ size } width={ size * 1.2 }`